cli/command/stack/swarm/services.go

```diff
@@ -3,6 +3,9 @@ package swarm
 import (
 	"context"
 	"fmt"
+	"sort"
+
+	"vbom.ml/util/sortorder"
```

Comment:
nit: we don't group imports in three groups (only two), so this should be grouped (and sorted) with the ones below.

Reply:
Changed.